### PR TITLE
Updated association modal to look like updated design

### PR DESCRIPTION
**Pull request by BraydonKains**
updated modal

### CODE REVIEW

**Comment by quinnbudan** [comment]
Something I just noticed when comparing your screenshot to the screenshot mockup in the story, there's a bit more of a margin between the info paragraph and "**Department ***". Other than that looks great!

**Review by quinnbudan — APPROVED** [review]

**Comment by BraydonKains** [comment]
Updated:
updated modal

**Review by StephaniePitman — APPROVED** [review]

File: src/components/dialog/association-dialog.js [hunk 1]
```diff
@@ -315,7 +340,7 @@ class TccAssociationDialog extends BaseMixin(LitElement) {
 
 	_renderDepartmentSelector() {
 		return html`
-			<div class="association_form__input_group">
+			<div class="input_group">
```
Comment (louiscoste) [inline src/components/dialog/association-dialog.js:343]: Why was this renamed? I liked the format we had

File: src/components/dialog/association-dialog.js [hunk 2]
```diff
@@ -110,18 +110,43 @@ class TccAssociationDialog extends BaseMixin(LitElement) {
...
+			.input_group {
+				margin-top: 12px;
 				margin-bottom: 24px;
 				display: flex;
 				flex-direction: column;
 			}
 
-			.association_form__button_group {
-				margin-top: 78px;
+			.xfix_group {
```
Comment (louiscoste) [inline src/components/dialog/association-dialog.js:120]: What does `xfix` mean?

File: src/components/dialog/association-dialog.js (hunk 1 above)
Comment (BraydonKains) [reply src/components/dialog/association-dialog.js:343]: Block Element Modifier usually means Block__Element--Modifier
`input_group` is not an Element, it's a Block. I also thought putting `association_form` everywhere wasn't terribly helpful since these styles are already local to this element, and so the fact that they are for the `association_form` is obvious.

File: src/components/dialog/association-dialog.js (hunk 2 above)
Comment (BraydonKains) [reply src/components/dialog/association-dialog.js:120]: "Pre"fix and "Suf"fix share these styles. I couldn't think of a better name.

File: src/components/dialog/association-dialog.js (hunk 1 above)
Comment (BraydonKains) [reply src/components/dialog/association-dialog.js:343]: BEM kinda falls apart when there are Blocks within Blocks though. Not as much a fan of this style after this.

**Review by louiscoste — APPROVED** [review]

**Review by quinnbudan — APPROVED** [review]

**Review by louiscoste — APPROVED** [review]
